tuxemon/resources/maps/route1.tmx

```diff
@@ -117,95 +122,80 @@
-    <property name="act2" value="wait_for_input K_RETURN"/>
-    <property name="act3" value="start_battle npc_karrianna"/>
-    <property name="act4" value="set_variable karrianna_battle_won:yes"/>
-    <property name="cond1" value="is player_facing left"/>
-    <property name="cond2" value="is player_at"/>
-    <property name="cond3" value="is variable_set karrianna_battle_won:started"/>
-    <property name="cond4" value="not variable_set karrianna_battle_won:yes"/>
-    <property name="cond5" value="not variable_set karrianna_battle_won:notyet"/>
+    <property name="act1" value="translated_dialog xero_hideout1"/>
+    <property name="cond1" value="is player_at"/>
+    <property name="cond2" value="is player_facing up"/>
+    <property name="cond3" value="is button_pressed K_RETURN"/>
    </properties>
   </object>
-  <object id="49" name="Karrianna's end" type="event" x="288" y="608" width="16" height="16">
+  <object id="62" name="blocking the way" type="event" x="448" y="512" width="16" height="160">
    <properties>
-    <property name="act1" value="translated_dialog karrianna_dialog2"/>
+    <property name="act1" value="translated_dialog xero_grunts_block"/>
     <property name="cond1" value="is player_at"/>
     <property name="cond2" value="is player_facing left"/>
-    <property name="cond3" value="is variable_set karrianna_battle_won:yes"/>
-    <property name="cond4" value="is button_pressed K_RETURN"/>
-    <property name="cond5" value="not variable_set karrianna_battle_won:started"/>
-    <property name="cond6" value="not variable_set karrianna_battle_won:notyet"/>
+    <property name="cond3" value="is button_pressed K_RETURN"/>
    </properties>
   </object>
-  <object id="57" name="Establish hideout" type="event" x="256" y="384" width="16" height="16">
+  <object id="68" name="talk to me" type="event" x="176" y="144" width="16" height="16">
    <properties>
-    <property name="act1" value="translated_dialog xero_hideout1"/>
-    <property name="cond1" value="is player_at"/>
-    <property name="cond2" value="is player_facing up"/>
-    <property name="cond3" value="is button_pressed K_RETURN"/>
+    <property name="act1" value="create_npc npc_xerogrund1,33,16,teamxerogrunt1,stand"/>
+    <property name="act2" value="create_npc npc_xerogrund2,34,16,teamxerogrunt1,stand"/>
+    <property name="cond1" value="not npc_exists npc_xerogrund1"/>
+    <property name="cond2" value="not npc_exists npc_xerogrund2"/>
    </properties>
   </object>
-  <object id="58" name="ask grunt" type="event" x="224" y="272" width="16" height="16">
+  <object id="69" name="talkin" type="event" x="528" y="272" width="16" height="16">
    <properties>
-    <property name="act1" value="translated_dialog xero_grunt1"/>
-    <property name="act2" value="translated_dialog xero_grunt2"/>
-    <property name="act3" value="translated_dialog xero_grunt3"/>
+    <property name="act1" value="translated_dialog waiting_for_misa"/>
```

Comment:
This dialog says to talk to the person to his left, which I think it a little confusing. Technically it is HIS left, but he's to the right of the NPC on the screen. Maybe we should change this dialog to say "Talk to my friend next to me?"

![screenshot](https://i.imgur.com/wnUJ6bT.png)

Reply:
You're right, this should be a quick fix